pimd/pim_nb_config.c

```diff
@@ -2353,30 +2354,30 @@ int routing_control_plane_protocols_control_plane_protocol_pim_address_family_rp
 		vrf = nb_running_get_entry(args->dnode, NULL, true);
 		pim = vrf->info;
 		yang_dnode_get_ip(&rp_addr, args->dnode, "./rp-address");
```

Comment:
I think we should add `yang_dnode_get_pimaddr` (in PIM code, not in `lib/`, because it is PIM specific)

(repeated a few times below)

Actually it could just be
```
#if PIM_IPV == 4
#define yang_dnode_get_pimaddr yang_dnode_get_ipv4
#else
#define yang_dnode_get_pimaddr yang_dnode_get_ipv6
#endif
```